src/app/components/Curation/index.tsx

```diff
@@ -158,7 +159,11 @@ export default ({
         !isLive()
       ) {
         return (
-          <PortraitVideoCarousel title={title} items={portraitVideo.items} />
+          <PortraitVideoCarousel
+            title={title}
+            items={portraitVideo.items}
+            {...(curationId && { curationId })}
```

Comment:
Whilst using `curationId` is fine for now, I wonder if this will become less relevant if we were to ever reuse the carousel on another page... Could we consider renaming the prop to `groupTrackingId` or something similar? (We have used `eventTrackingData` as a prop before, so it's OK to include tracking specific props here)